src/SwapToken.sol

```diff
@@ -0,0 +1,42 @@
+import "@uniswap/v3-periphery/contracts/libraries/TransferHelper.sol";
+import "@uniswap/v3-periphery/contracts/interfaces/ISwapRouter.sol";
+import "./PriceFeedConsumer.sol";
+
+contract SwapToken is Ownable {
```

Comment:
We don't need this contract, we can just put all of this logic inside `Portfolio.sol`